src/llvm-coverage-instrumentation.md

```diff
@@ -338,16 +338,15 @@ with the following steps:
 1. [`generate_coverage_spans()`][generate-coverage-spans] computes the minimum set of distinct,
    non-branching code regions, from the MIR. These `CoverageSpan`s
    represent a span of code that must be counted.
-2. [`make_bcb_counters()`][make-bcb-counters] generates `CoverageKind::Counter`s and
-   `CoverageKind::Expression`s for each `CoverageSpan`, plus additional
-   `intermediate_expressions`[^intermediate-expressions], not associated with any `CodeRegion`, but
+2. [`make_bcb_counters()`][make-bcb-counters] generates `BcbCounter::Counter`s and
+   `BcbCounter::Expression`s for each `CoverageSpan`, plus additional
+   _intermediate expressions_[^intermediate-expressions] that are not associated
+   with any `CodeRegion`, but
    are required to compute a final `Expression` value for a `CodeRegion`.
 3. Inject the new counters into the MIR, as new `StatementKind::Coverage`
-   statements. This is done by three distinct functions:
-   - `inject_coverage_span_counters()`
-   - `inject_indirect_counters()`
-   - `inject_intermediate_expression()`, called for each intermediate expression
-     returned from `make_bcb_counters()`
```

Comment:
The three `inject` function mentioned here were removed by https://github.com/rust-lang/rust/pull/116917, and replaced with a much simpler pair of loops in `create_mappings_and_inject_coverage_statements`.